install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -0,0 +1,425 @@
+certificates to use during the other phases of this setup. We will start by
+using the auto-generated certificates created as part of running:
++
+====
+----
+# openshift start \
+    --public-master=https://openshift.example.com:8443 \
+    --write-config=/etc/origin/
+----
+====
++
+Among other things, this generates a *_/etc/origin/master/ca.{cert|key}_*. Use
+this signing certificate to generate keys to use on the authenticating proxy.
++
+====
+----
+# mkdir -p /etc/origin/proxy/
+# oadm ca create-server-cert \
+    --cert='/etc/origin/proxy/proxy.example.com.crt' \
+    --key='/etc/origin/proxy/proxy.example.com.key' \
+    --hostnames=proxy.example.com \
+    --signer-cert=/etc/origin/master/ca.crt \
+    --signer-key='/etc/origin/master/ca.key' \
+    --signer-serial='/etc/origin/master/ca.serial.txt'
+----
+====
++
+This command automatically adds all valid IP addresses.
++
+[IMPORTANT]
+====
+Ensure that any host names that might need to access the proxy are listed, or
+the HTTPS connection will fail.
+====
+
+. Generate the API client certificate that the authenticating proxy will use
+to prove its identity to {product-title}. This is necessary and prevents
+malicious users from impersonating the proxy and sending fake identities.
+
+. Create a new CA to sign this client certificate:
++
+====
+----
+# oadm ca create-signer-cert \
+  --cert='/etc/origin/proxy/proxyca.crt' \
+  --key='/etc/origin/proxy/proxyca.key' \
+  --name='openshift-proxy-signer@`date +%s`' \
+  --serial='/etc/origin/proxy/proxyca.serial.txt'
+----
+====
++
+`date +%s` makes the signer unique. You can use any name you prefer, however.
++
+====
+----
+# oadm create-api-client-config \
+    --certificate-authority='/etc/origin/proxy/proxyca.crt' \
+    --client-dir='/etc/origin/proxy' \
+    --signer-cert='/etc/origin/proxy/proxyca.crt' \
+    --signer-key='/etc/origin/proxy/proxyca.key' \
+    --signer-serial='/etc/origin/proxy/proxyca.serial.txt' \
+    --user='system:proxy'
+# cat /etc/origin/proxy/system\:proxy.crt \
+      /etc/origin/proxy/system\:proxy.key \
+      > /etc/origin/proxy/authproxy.pem
+----
+====
+
+[[sssd-phase-2-authenticating-proxy-setup]]
+== Phase 2: Authenticating Proxy Setup
+
+This section guides you through the steps to authenticate the proxy setup.
+
+[[phase-2-step-1-copy-certificates]]
+=== Step 1: Copy Certificates
+
+From *_openshift.example.com_*, securely copy the necessary certificates to the
+proxy machine:
+
+====
+----
+# scp /etc/origin/proxy/master/ca.crt \
+      root@proxy.example.com:/etc/pki/CA/certs/
+
+# scp /etc/origin/proxy/proxy.example.com.crt \
+      /etc/origin/proxy/authproxy.pem \
+      root@proxy.example.com:/etc/pki/tls/certs/
+
+# scp /etc/origin/proxy/proxy.example.com.key \
+      root@proxy.example.com:/etc/pki/tls/private/
+----
+====
+
+[[phase-2-step-2-sssd-configuration]]
+=== Step 2: SSSD Configuration
+. Install a new VM with a recent operating system. Run SSSD 1.12.0 or later so
```

Comment:
Maybe rephrase this to "Install a new VM with an operating system that include SSSD 1.12.0 or later so that you can use the mod_identity_lookup module."